src/Dialog/Dialog.js

```diff
@@ -29,9 +29,13 @@ export const styleSheet = createStyleSheet('MuiDialog', theme => ({
   },
   paperWidthXs: {
     maxWidth: theme.breakpoints.getWidth('xs'),
+    marginLeft: '40px',
+    marginRight: '40px',
   },
   paperWidthSm: {
     maxWidth: theme.breakpoints.getWidth('sm'),
+    marginLeft: '40px',
```

Comment:
Why scoping the style to this classes?
I think that the best fix would be the following:
```diff
  paper: {
+   margin: theme.spacing.unit * 4,
    display: 'flex',
```